### ISSUE TITLE
shutter when I press select area gets stuck

### ISSUE THREAD

**Issue by lasermtv07** [shutter-project/shutter#580]
#### Brief summary of issue
chi, when I press select area, shutter gets just stuck on a grey screen with text 'Desktop @ QReact(0,0 1920x1080) - Plasma in the middle
I use arch btw and KDE Plasma DE. Other modes (like screenshot one window or screenshot full screen dont work at all)
#### Steps to reproduce the issue

1.  press for instance Ctrl+PrScr (which I use to take screenshot)
2. repeat several times it wont do anything
3. see the result and give up.

#### Error output
[misa@E ~]$ shutter -v Subroutine Pango::Layout::set_text redefined at /usr/share/perl5/vendor_perl/Gtk3.pm line 2299.         require Gtk3.pm called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72 Subroutine Pango::Layout::set_markup redefined at /usr/share/perl5/vendor_perl/Gtk3.pm line 2305.         require Gtk3.pm called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72 GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed at /usr/lib/perl5/5.36/vendor_perl/Glib/Object/Introspection.pm line 110.  at /usr/share/perl5/vendor_perl/Gtk3.pm line 489.         Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72 GLib-CRITICAL **: g_once_init_leave: assertion 'result != 0' failed at /usr/lib/perl5/5.36/vendor_perl/Glib/Object/Introspection.pm line 110.  at /usr/share/perl5/vendor_perl/Gtk3.pm line 489.         Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72 GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed at /usr/lib/perl5/5.36/vendor_perl/Glib/Object/Introspection.pm line 110.  at /usr/share/perl5/vendor_perl/Gtk3.pm line 489.         Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72 GLib-CRITICAL **: g_once_init_leave: assertion 'result != 0' failed at /usr/lib/perl5/5.36/vendor_perl/Glib/Object/Introspection.pm line 110.  at /usr/share/perl5/vendor_perl/Gtk3.pm line 489.         Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72 GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed at /usr/lib/perl5/5.36/vendor_perl/Glib/Object/Introspection.pm line 110.  at /usr/share/perl5/vendor_perl/Gtk3.pm line 489.         Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72 GLib-CRITICAL **: g_once_init_leave: assertion 'result != 0' failed at /usr/lib/perl5/5.36/vendor_perl/Glib/Object/Introspection.pm line 110.  at /usr/share/perl5/vendor_perl/Gtk3.pm line 489.         Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72         Shutter::App::BEGIN() called at /usr/bin/shutter line 72         eval {...} called at /usr/bin/shutter line 72  (shutter:24401): Gtk-WARNING **: 19:43:33.973: Theme parsing error: gtk.css:1649:16: '-gtk-icon-size' is not a valid property name  (shutter:24401): Gtk-WARNING **: 19:43:33.973: Theme parsing error: gtk.css:1652:16: '-gtk-icon-size' is not a valid property name Variable "$progname_active" will not stay shared at /usr/bin/shutter line 2751. Variable "$progname" will not stay shared at /usr/bin/shutter line 2752. Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 2760. Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 2761. Variable "$trans_check" will not stay shared at /usr/bin/shutter line 2771. Variable "$css_provider_alpha" will not stay shared at /usr/bin/shutter line 2772. Variable "$trans_custom" will not stay shared at /usr/bin/shutter line 2777. Variable "$trans_custom_btn" will not stay shared at /usr/bin/shutter line 2778. Variable "$trans_backg" will not stay shared at /usr/bin/shutter line 2784. Variable "$window" will not stay shared at /usr/bin/shutter line 2787. Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 2792. Variable "$cursor_status_active" will not stay shared at /usr/bin/shutter line 2792. Variable "$delay_status" will not stay shared at /usr/bin/shutter line 2802. Variable "$delay" will not stay shared at /usr/bin/shutter line 2802. Variable "$delay_vlabel" will not stay shared at /usr/bin/shutter line 2803. Variable "$d" will not stay shared at /usr/bin/shutter line 2803. Variable "$delay_status_vlabel" will not stay shared at /usr/bin/shutter line 2809. Variable "$menu_delay_vlabel" will not stay shared at /usr/bin/shutter line 2814. Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 2814. Variable "$hide_time_vlabel" will not stay shared at /usr/bin/shutter line 2819. Variable "$hide_time" will not stay shared at /usr/bin/shutter line 2819. Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 2824. Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 2825. Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 2840. Variable "$scale" will not stay shared at /usr/bin/shutter line 2841. Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 2853. Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 2854. Variable "$bordereffect_vlabel" will not stay shared at /usr/bin/shutter line 2862. Variable "$scale_label" will not stay shared at /usr/bin/shutter line 2869. Variable "$combobox_ns" will not stay shared at /usr/bin/shutter line 2893. Variable "$window" will not stay shared at /usr/bin/shutter line 2910. Variable "$hide_active" will not stay shared at /usr/bin/shutter line 2914. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 2918. Variable "$x11_supported" will not stay shared at /usr/bin/shutter line 2943. Variable "$d" will not stay shared at /usr/bin/shutter line 2945. Variable "$notify_ptimeout_active" will not stay shared at /usr/bin/shutter line 2969. Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 2971. Variable "$hide_time" will not stay shared at /usr/bin/shutter line 3036. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3059. Variable "$notebook" will not stay shared at /usr/bin/shutter line 3061. Variable "$lp" will not stay shared at /usr/bin/shutter line 3079. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3108. Variable "$close_at_close_active" will not stay shared at /usr/bin/shutter line 3123. Variable "$tray" will not stay shared at /usr/bin/shutter line 3124. Variable "$window" will not stay shared at /usr/bin/shutter line 3126. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 3127. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3143. Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 3158. Variable "$sas" will not stay shared at /usr/bin/shutter line 3162. Variable "$fs_active" will not stay shared at /usr/bin/shutter line 3164. Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 3165. Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 3166. Variable "$app" will not stay shared at /usr/bin/shutter line 3169. Variable "$window" will not stay shared at /usr/bin/shutter line 3201. Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 3210. Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 3228. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 3248. Variable "$d" will not stay shared at /usr/bin/shutter line 3260. Variable "$sm" will not stay shared at /usr/bin/shutter line 3274. Variable "$sm" will not stay shared at /usr/bin/shutter line 3299. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3330. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3364. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3388. Variable "$clipboard" will not stay shared at /usr/bin/shutter line 3425. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3461. Variable "$notebook" will not stay shared at /usr/bin/shutter line 3470. Variable "$sm" will not stay shared at /usr/bin/shutter line 3480. Variable "$st" will not stay shared at /usr/bin/shutter line 3490. Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 3503. Variable "$notebook" will not stay shared at /usr/bin/shutter line 3691. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3691. Variable "$settings_dialog" will not stay shared at /usr/bin/shutter line 3699. Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3715. Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 3715. Variable "$sas" will not stay shared at /usr/bin/shutter line 3729. Variable "$fs_active" will not stay shared at /usr/bin/shutter line 3731. Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 3732. Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 3733. Variable "$pagesetup" will not stay shared at /usr/bin/shutter line 3752. Variable "$window" will not stay shared at /usr/bin/shutter line 3752. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3774. Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3816. Variable "$sas" will not stay shared at /usr/bin/shutter line 3829. Variable "$fs_active" will not stay shared at /usr/bin/shutter line 3831. Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 3832. Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 3833. Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3852. Variable "$d" will not stay shared at /usr/bin/shutter line 3859. Variable "$sd" will not stay shared at /usr/bin/shutter line 3861. Variable "$settings_xml" will not stay shared at /usr/bin/shutter line 3872. Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3873. Variable "$d" will not stay shared at /usr/bin/shutter line 3880. Variable "$tray_legacy" will not stay shared at /usr/bin/shutter line 3892. Variable "$tray_libappindicator" will not stay shared at /usr/bin/shutter line 3899. Variable "$tray" will not stay shared at /usr/bin/shutter line 3903. Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 3921. Variable "$notebook" will not stay shared at /usr/bin/shutter line 3936. Variable "$d" will not stay shared at /usr/bin/shutter line 3956. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3962. Variable "$d" will not stay shared at /usr/bin/shutter line 3976. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 4001. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 4034. Variable "$notebook" will not stay shared at /usr/bin/shutter line 4064. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 4105. Variable "$sd" will not stay shared at /usr/bin/shutter line 4152. Variable "$d" will not stay shared at /usr/bin/shutter line 4152. Variable "@signal_connections" will not stay shared at /usr/bin/shutter line 4168. Variable "$app" will not stay shared at /usr/bin/shutter line 4169. Variable "$tray" will not stay shared at /usr/bin/shutter line 4175. Variable "$st" will not stay shared at /usr/bin/shutter line 4210. Variable "$sm" will not stay shared at /usr/bin/shutter line 4210. Variable "$x11_supported" will not stay shared at /usr/bin/shutter line 4214. Variable "$window" will not stay shared at /usr/bin/shutter line 4259. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 4271. Variable "$st" will not stay shared at /usr/bin/shutter line 4277. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 4311. Variable "$css_provider_alpha" will not stay shared at /usr/bin/shutter line 4314. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 4401. Variable "%settings" will not stay shared at /usr/bin/shutter line 4488. Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 4490. Variable "$sm" will not stay shared at /usr/bin/shutter line 4494. Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 4502. Variable "$scale" will not stay shared at /usr/bin/shutter line 4503. Variable "$filename" will not stay shared at /usr/bin/shutter line 4504. Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 4505. Variable "$save_auto_active" will not stay shared at /usr/bin/shutter line 4509. Variable "$save_ask_active" will not stay shared at /usr/bin/shutter line 4510. Variable "$save_no_active" will not stay shared at /usr/bin/shutter line 4511. Variable "$image_autocopy_active" will not stay shared at /usr/bin/shutter line 4512. Variable "$fname_autocopy_active" will not stay shared at /usr/bin/shutter line 4513. Variable "$no_autocopy_active" will not stay shared at /usr/bin/shutter line 4514. Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 4515. Variable "$delay" will not stay shared at /usr/bin/shutter line 4516. Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 4519. Variable "$st" will not stay shared at /usr/bin/shutter line 4523. Variable "$progname" will not stay shared at /usr/bin/shutter line 4536. Variable "$progname_active" will not stay shared at /usr/bin/shutter line 4545. Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 4546. Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 4547. Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 4548. Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 4549. Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 4550. Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 4551. Variable "$bordereffect_cbtn" will not stay shared at /usr/bin/shutter line 4552. Variable "$zoom_active" will not stay shared at /usr/bin/shutter line 4556. Variable "$as_help_active" will not stay shared at /usr/bin/shutter line 4557. Variable "$asel_size3" will not stay shared at /usr/bin/shutter line 4558. Variable "$asel_size4" will not stay shared at /usr/bin/shutter line 4559. Variable "$asel_size1" will not stay shared at /usr/bin/shutter line 4560. Variable "$asel_size2" will not stay shared at /usr/bin/shutter line 4561. Variable "$border_active" will not stay shared at /usr/bin/shutter line 4562. Variable "$winresize_active" will not stay shared at /usr/bin/shutter line 4563. Variable "$winresize_w" will not stay shared at /usr/bin/shutter line 4564. Variable "$winresize_h" will not stay shared at /usr/bin/shutter line 4565. Variable "$autoshape_active" will not stay shared at /usr/bin/shutter line 4566. Variable "$visible_windows_active" will not stay shared at /usr/bin/shutter line 4567. Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 4568. Variable "$menu_waround_active" will not stay shared at /usr/bin/shutter line 4569. Variable "$combobox_web_width" will not stay shared at /usr/bin/shutter line 4570. Variable "$trans_check" will not stay shared at /usr/bin/shutter line 4573. Variable "$trans_custom" will not stay shared at /usr/bin/shutter line 4574. Variable "$trans_custom_btn" will not stay shared at /usr/bin/shutter line 4575. Variable "$trans_backg" will not stay shared at /usr/bin/shutter line 4577. Variable "$session_asc" will not stay shared at /usr/bin/shutter line 4579. Variable "$session_asc_combo" will not stay shared at /usr/bin/shutter line 4580. Variable "$session_desc" will not stay shared at /usr/bin/shutter line 4581. Variable "$session_desc_combo" will not stay shared at /usr/bin/shutter line 4582. Variable "$fs_active" will not stay shared at /usr/bin/shutter line 4585. Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 4586. Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 4587. Variable "$hide_active" will not stay shared at /usr/bin/shutter line 4588. Variable "$hide_time" will not stay shared at /usr/bin/shutter line 4589. Variable "$present_after_active" will not stay shared at /usr/bin/shutter line 4590. Variable "$close_at_close_active" will not stay shared at /usr/bin/shutter line 4591. Variable "$notify_after_active" will not stay shared at /usr/bin/shutter line 4592. Variable "$notify_timeout_active" will not stay shared at /usr/bin/shutter line 4593. Variable "$notify_ptimeout_active" will not stay shared at /usr/bin/shutter line 4594. Variable "$combobox_ns" will not stay shared at /usr/bin/shutter line 4595. Variable "$ask_on_delete_active" will not stay shared at /usr/bin/shutter line 4596. Variable "$delete_on_close_active" will not stay shared at /usr/bin/shutter line 4597. Variable "$ask_on_fs_delete_active" will not stay shared at /usr/bin/shutter line 4598. Variable "$ftp_remote_entry" will not stay shared at /usr/bin/shutter line 4601. Variable "$ftp_mode_combo" will not stay shared at /usr/bin/shutter line 4602. Variable "$ftp_username_entry" will not stay shared at /usr/bin/shutter line 4603. Variable "$ftp_password_entry" will not stay shared at /usr/bin/shutter line 4604. Variable "$ftp_wurl_entry" will not stay shared at /usr/bin/shutter line 4605. Variable "%plugins" will not stay shared at /usr/bin/shutter line 4608. Variable "$sd" will not stay shared at /usr/bin/shutter line 4633. Variable "$d" will not stay shared at /usr/bin/shutter line 4633. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 4641. Variable "%accounts" will not stay shared at /usr/bin/shutter line 4667. Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 4717. Variable "$int_jpeg" will not stay shared at /usr/bin/shutter line 4717. Variable "$int_png" will not stay shared at /usr/bin/shutter line 4719. Variable "$scale" will not stay shared at /usr/bin/shutter line 4728. Variable "$filename" will not stay shared at /usr/bin/shutter line 4730. Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 4733. Variable "$save_auto_active" will not stay shared at /usr/bin/shutter line 4735. Variable "$save_ask_active" will not stay shared at /usr/bin/shutter line 4736. Variable "$save_no_active" will not stay shared at /usr/bin/shutter line 4737. Variable "$image_autocopy_active" will not stay shared at /usr/bin/shutter line 4739. Variable "$fname_autocopy_active" will not stay shared at /usr/bin/shutter line 4740. Variable "$no_autocopy_active" will not stay shared at /usr/bin/shutter line 4741. Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 4743. Variable "$delay" will not stay shared at /usr/bin/shutter line 4744. Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 4753. Variable "$st" will not stay shared at /usr/bin/shutter line 4756. Variable "$progname" will not stay shared at /usr/bin/shutter line 4771. Variable "$progname_active" will not stay shared at /usr/bin/shutter line 4774. Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 4776. Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 4777. Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 4779. Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 4780. Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 4782. Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 4783. Variable "$bordereffect_cbtn" will not stay shared at /usr/bin/shutter line 4785. Variable "$zoom_active" will not stay shared at /usr/bin/shutter line 4789. Variable "$as_help_active" will not stay shared at /usr/bin/shutter line 4791. Variable "$asel_size3" will not stay shared at /usr/bin/shutter line 4793. Variable "$asel_size4" will not stay shared at /usr/bin/shutter line 4794. Variable "$asel_size1" will not stay shared at /usr/bin/shutter line 4795. Variable "$asel_size2" will not stay shared at /usr/bin/shutter line 4796. Variable "$border_active" will not stay shared at /usr/bin/shutter line 4798. Variable "$winresize_active" will not stay shared at /usr/bin/shutter line 4800. Variable "$winresize_w" will not stay shared at /usr/bin/shutter line 4801. Variable "$winresize_h" will not stay shared at /usr/bin/shutter line 4802. Variable "$autoshape_active" will not stay shared at /usr/bin/shutter line 4804. Variable "$visible_windows_active" will not stay shared at /usr/bin/shutter line 4805. Variable "$menu_waround_active" will not stay shared at /usr/bin/shutter line 4806. Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 4807. Variable "$combobox_web_width" will not stay shared at /usr/bin/shutter line 4808. Variable "$trans_check" will not stay shared at /usr/bin/shutter line 4811. Variable "$trans_custom" will not stay shared at /usr/bin/shutter line 4812. Variable "$trans_custom_btn" will not stay shared at /usr/bin/shutter line 4814. Variable "$trans_backg" will not stay shared at /usr/bin/shutter line 4816. Variable "$session_asc" will not stay shared at /usr/bin/shutter line 4818. Variable "$session_asc_combo" will not stay shared at /usr/bin/shutter line 4819. Variable "$session_desc" will not stay shared at /usr/bin/shutter line 4820. Variable "$session_desc_combo" will not stay shared at /usr/bin/shutter line 4821. Variable "$fs_active" will not stay shared at /usr/bin/shutter line 4824. Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 4825. Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 4826. Variable "$hide_active" will not stay shared at /usr/bin/shutter line 4827. Variable "$hide_time" will not stay shared at /usr/bin/shutter line 4828. Variable "$present_after_active" will not stay shared at /usr/bin/shutter line 4829. Variable "$close_at_close_active" will not stay shared at /usr/bin/shutter line 4830. Variable "$notify_after_active" will not stay shared at /usr/bin/shutter line 4831. Variable "$notify_timeout_active" will not stay shared at /usr/bin/shutter line 4832. Variable "$notify_ptimeout_active" will not stay shared at /usr/bin/shutter line 4833. Variable "$combobox_ns" will not stay shared at /usr/bin/shutter line 4834. Variable "$ask_on_delete_active" will not stay shared at /usr/bin/shutter line 4835. Variable "$delete_on_close_active" will not stay shared at /usr/bin/shutter line 4836. Variable "$ask_on_fs_delete_active" will not stay shared at /usr/bin/shutter line 4837. Variable "$ftp_remote_entry" will not stay shared at /usr/bin/shutter line 4846. Variable "$ftp_mode_combo" will not stay shared at /usr/bin/shutter line 4847. Variable "$ftp_username_entry" will not stay shared at /usr/bin/shutter line 4848. Variable "$ftp_password_entry" will not stay shared at /usr/bin/shutter line 4849. Variable "$ftp_wurl_entry" will not stay shared at /usr/bin/shutter line 4850. Variable "$accounts_tree" will not stay shared at /usr/bin/shutter line 4864. Variable "$accounts_model" will not stay shared at /usr/bin/shutter line 4866. Variable "%plugins" will not stay shared at /usr/bin/shutter line 4904. Variable "$sd" will not stay shared at /usr/bin/shutter line 4920. Variable "$d" will not stay shared at /usr/bin/shutter line 4920. Variable "$d" will not stay shared at /usr/bin/shutter line 4949. Variable "$sd" will not stay shared at /usr/bin/shutter line 5000. Variable "$d" will not stay shared at /usr/bin/shutter line 5000. Variable "%accounts" will not stay shared at /usr/bin/shutter line 5010. Variable "$status" will not stay shared at /usr/bin/shutter line 5084. Variable "$d" will not stay shared at /usr/bin/shutter line 5096. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5113. Variable "$notebook" will not stay shared at /usr/bin/shutter line 5118. Variable "$status" will not stay shared at /usr/bin/shutter line 5172. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5176. Variable "$d" will not stay shared at /usr/bin/shutter line 5192. Variable "$sd" will not stay shared at /usr/bin/shutter line 5207. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5218. Variable "$d" will not stay shared at /usr/bin/shutter line 5219. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5229. Variable "$ask_on_delete_active" will not stay shared at /usr/bin/shutter line 5248. Variable "$sd" will not stay shared at /usr/bin/shutter line 5249. Variable "$d" will not stay shared at /usr/bin/shutter line 5250. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5250. Variable "$notebook" will not stay shared at /usr/bin/shutter line 5265. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5270. Variable "$window" will not stay shared at /usr/bin/shutter line 5280. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 5280. Variable "$delete_on_close_active" will not stay shared at /usr/bin/shutter line 5378. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5383. Variable "$notebook" will not stay shared at /usr/bin/shutter line 5389. Variable "$d" will not stay shared at /usr/bin/shutter line 5390. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5394. Variable "$window" will not stay shared at /usr/bin/shutter line 5404. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 5404. Variable "$clipboard" will not stay shared at /usr/bin/shutter line 5481. Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 5485. Variable "$sd" will not stay shared at /usr/bin/shutter line 5488. Variable "$d" will not stay shared at /usr/bin/shutter line 5488. Variable "$sp" will not stay shared at /usr/bin/shutter line 5498. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5527. Variable "$lp" will not stay shared at /usr/bin/shutter line 5548. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5548. Variable "$clipboard" will not stay shared at /usr/bin/shutter line 5561. Variable "$d" will not stay shared at /usr/bin/shutter line 5562. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5586. Variable "$filename" will not stay shared at /usr/bin/shutter line 5598. Variable "$filename_hint" will not stay shared at /usr/bin/shutter line 5599. Variable "$d" will not stay shared at /usr/bin/shutter line 5613. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5635. Variable "$d" will not stay shared at /usr/bin/shutter line 5651. Variable "$sd" will not stay shared at /usr/bin/shutter line 5658. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5687. Variable "$d" will not stay shared at /usr/bin/shutter line 5702. Variable "$sd" will not stay shared at /usr/bin/shutter line 5709. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5730. Variable "%plugins" will not stay shared at /usr/bin/shutter line 5746. Variable "$sd" will not stay shared at /usr/bin/shutter line 5747. Variable "$d" will not stay shared at /usr/bin/shutter line 5747. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5756. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5783. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5789. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5823. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5828. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5859. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5876. Variable "$scale" will not stay shared at /usr/bin/shutter line 5893. Variable "$filename" will not stay shared at /usr/bin/shutter line 5902. Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 5908. Variable "$save_no_active" will not stay shared at /usr/bin/shutter line 5909. Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 5920. Variable "$sd" will not stay shared at /usr/bin/shutter line 5923. Variable "$d" will not stay shared at /usr/bin/shutter line 5923. Variable "$delay" will not stay shared at /usr/bin/shutter line 5938. Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 5954. Variable "$notify_timeout_active" will not stay shared at /usr/bin/shutter line 5966. Variable "$wnck_screen" will not stay shared at /usr/bin/shutter line 5967. Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 5968. Variable "$border_active" will not stay shared at /usr/bin/shutter line 5999. Variable "$winresize_active" will not stay shared at /usr/bin/shutter line 5999. Variable "$winresize_w" will not stay shared at /usr/bin/shutter line 5999. Variable "$winresize_h" will not stay shared at /usr/bin/shutter line 5999. Variable "$hide_time" will not stay shared at /usr/bin/shutter line 6000. Variable "$autoshape_active" will not stay shared at /usr/bin/shutter line 6000. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 6010. Variable "$visible_windows_active" will not stay shared at /usr/bin/shutter line 6011. Variable "$menu_waround_active" will not stay shared at /usr/bin/shutter line 6011. Variable "$zoom_active" will not stay shared at /usr/bin/shutter line 6038. Variable "$as_help_active" will not stay shared at /usr/bin/shutter line 6038. Variable "$asel_size3" will not stay shared at /usr/bin/shutter line 6038. Variable "$asel_size4" will not stay shared at /usr/bin/shutter line 6039. Variable "$asel_size1" will not stay shared at /usr/bin/shutter line 6039. Variable "$asel_size2" will not stay shared at /usr/bin/shutter line 6039. Variable "$combobox_web_width" will not stay shared at /usr/bin/shutter line 6050. Variable "$st" will not stay shared at /usr/bin/shutter line 6058. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6198. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6211. Variable "$present_after_active" will not stay shared at /usr/bin/shutter line 6286. Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 6391. Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 6397. Variable "$bordereffect_cbtn" will not stay shared at /usr/bin/shutter line 6397. Variable "$save_ask_active" will not stay shared at /usr/bin/shutter line 6401. Variable "$sp" will not stay shared at /usr/bin/shutter line 6427. Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 6439. Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 6441. Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 6454. Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 6457. Variable "$lp" will not stay shared at /usr/bin/shutter line 6461. Variable "$no_autocopy_active" will not stay shared at /usr/bin/shutter line 6507. Variable "$image_autocopy_active" will not stay shared at /usr/bin/shutter line 6510. Variable "$clipboard" will not stay shared at /usr/bin/shutter line 6511. Variable "$fname_autocopy_active" will not stay shared at /usr/bin/shutter line 6515. Variable "$progname_active" will not stay shared at /usr/bin/shutter line 6522. Variable "$progname" will not stay shared at /usr/bin/shutter line 6523. Variable "$notify_after_active" will not stay shared at /usr/bin/shutter line 6537. Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 6592. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6616. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6648. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6654. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6678. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6684. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6705. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6711. Variable "$pagesetup" will not stay shared at /usr/bin/shutter line 6724. Variable "$lp" will not stay shared at /usr/bin/shutter line 6750. Variable "$window" will not stay shared at /usr/bin/shutter line 6767. Variable "$d" will not stay shared at /usr/bin/shutter line 6792. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6792. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6808. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6840. Variable "$window" will not stay shared at /usr/bin/shutter line 6851. Variable "$sd" will not stay shared at /usr/bin/shutter line 6870. Variable "$d" will not stay shared at /usr/bin/shutter line 6870. Variable "$status" will not stay shared at /usr/bin/shutter line 6954. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6955. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6994. Variable "$window" will not stay shared at /usr/bin/shutter line 7002. Variable "$status" will not stay shared at /usr/bin/shutter line 7007. Variable "$d" will not stay shared at /usr/bin/shutter line 7008. Variable "$hu" will not stay shared at /usr/bin/shutter line 7008. Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 7024. Variable "$tray" will not stay shared at /usr/bin/shutter line 7025. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7045. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7066. Variable "$notebook" will not stay shared at /usr/bin/shutter line 7089. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7095. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7109. Variable "$notebook" will not stay shared at /usr/bin/shutter line 7119. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7125. Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 7144. Variable "$sm" will not stay shared at /usr/bin/shutter line 7153. Variable "$combobox_status_profiles" will not stay shared at /usr/bin/shutter line 7159. Variable "$combobox_status_profiles_label" will not stay shared at /usr/bin/shutter line 7162. Variable "$d" will not stay shared at /usr/bin/shutter line 7168. Variable "$status" will not stay shared at /usr/bin/shutter line 7170. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7230. Variable "$lp_ne" will not stay shared at /usr/bin/shutter line 7289. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 7457. Variable "$ask_on_fs_delete_active" will not stay shared at /usr/bin/shutter line 7486. Variable "$sd" will not stay shared at /usr/bin/shutter line 7493. Variable "$d" will not stay shared at /usr/bin/shutter line 7494. Variable "$notebook" will not stay shared at /usr/bin/shutter line 7502. Variable "$sp" will not stay shared at /usr/bin/shutter line 7548. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7686. Variable "$lp" will not stay shared at /usr/bin/shutter line 7702. Variable "$progname" will not stay shared at /usr/bin/shutter line 7711. Variable "$sd" will not stay shared at /usr/bin/shutter line 7752. Variable "$d" will not stay shared at /usr/bin/shutter line 7752. Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 7768. Variable "$sd" will not stay shared at /usr/bin/shutter line 7883. Variable "$d" will not stay shared at /usr/bin/shutter line 7884. Variable "$window" will not stay shared at /usr/bin/shutter line 7919. Variable "$d" will not stay shared at /usr/bin/shutter line 7919. Variable "$lp" will not stay shared at /usr/bin/shutter line 7947. Variable "%plugins" will not stay shared at /usr/bin/shutter line 7965. Variable "$window" will not stay shared at /usr/bin/shutter line 8066. Variable "$d" will not stay shared at /usr/bin/shutter line 8066. Variable "%accounts" will not stay shared at /usr/bin/shutter line 8106. Variable "$progname" will not stay shared at /usr/bin/shutter line 8207. Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 8218. Variable "$d" will not stay shared at /usr/bin/shutter line 8218. Variable "$settings_xml" will not stay shared at /usr/bin/shutter line 8323. Variable "$wnck_screen" will not stay shared at /usr/bin/shutter line 8356. Variable "$lp" will not stay shared at /usr/bin/shutter line 8366. Variable "$d" will not stay shared at /usr/bin/shutter line 8369. Variable "$d" will not stay shared at /usr/bin/shutter line 8417. Variable "$x11_supported" will not stay shared at /usr/bin/shutter line 8418. Variable "$lp" will not stay shared at /usr/bin/shutter line 8427. Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 8543. Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 8624. Variable "$sm" will not stay shared at /usr/bin/shutter line 8636. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 8642. Variable "$d" will not stay shared at /usr/bin/shutter line 8722. Variable "$settings_xml" will not stay shared at /usr/bin/shutter line 8779. Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 8820. Variable "$accounts_model" will not stay shared at /usr/bin/shutter line 8838. Variable "%accounts" will not stay shared at /usr/bin/shutter line 8843. Variable "%plugins" will not stay shared at /usr/bin/shutter line 8863. Variable "$lp" will not stay shared at /usr/bin/shutter line 8880. Variable "$d" will not stay shared at /usr/bin/shutter line 8912. Variable "%accounts" will not stay shared at /usr/bin/shutter line 8937. Variable "$accounts_model" will not stay shared at /usr/bin/shutter line 8942. Variable "$d" will not stay shared at /usr/bin/shutter line 9033. Variable "$window" will not stay shared at /usr/bin/shutter line 9259. Variable "$d" will not stay shared at /usr/bin/shutter line 9261. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 9265. Variable "$sd" will not stay shared at /usr/bin/shutter line 9352. Variable "$d" will not stay shared at /usr/bin/shutter line 9471. Variable "$window" will not stay shared at /usr/bin/shutter line 9471. Variable "$lp_ne" will not stay shared at /usr/bin/shutter line 9487. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 9529. Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 9530. Variable "$sm" will not stay shared at /usr/bin/shutter line 9554. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 9576. Variable "$d" will not stay shared at /usr/bin/shutter line 9579. Variable "$window" will not stay shared at /usr/bin/shutter line 9580. Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 9596. Variable "$lp_ne" will not stay shared at /usr/bin/shutter line 9612. Variable "$sp" will not stay shared at /usr/bin/shutter line 9768. Variable "$sd" will not stay shared at /usr/bin/shutter line 9817. Variable "$d" will not stay shared at /usr/bin/shutter line 9910. Variable "$window" will not stay shared at /usr/bin/shutter line 9910. Variable "%plugins" will not stay shared at /usr/bin/shutter line 9930. Variable "$lp" will not stay shared at /usr/bin/shutter line 10016. Variable "$sd" will not stay shared at /usr/bin/shutter line 10055. Variable "$d" will not stay shared at /usr/bin/shutter line 10105. Variable "$window" will not stay shared at /usr/bin/shutter line 10106. Variable "%accounts" will not stay shared at /usr/bin/shutter line 10116. Variable "$ftp_remote_entry" will not stay shared at /usr/bin/shutter line 10217. Variable "$ftp_mode_combo" will not stay shared at /usr/bin/shutter line 10231. Variable "$ftp_username_entry" will not stay shared at /usr/bin/shutter line 10243. Variable "$ftp_password_entry" will not stay shared at /usr/bin/shutter line 10257. Variable "$ftp_wurl_entry" will not stay shared at /usr/bin/shutter line 10269. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10395. Variable "$clipboard" will not stay shared at /usr/bin/shutter line 10496. Variable "$d" will not stay shared at /usr/bin/shutter line 10697. Variable "$d" will not stay shared at /usr/bin/shutter line 10717. Variable "$window" will not stay shared at /usr/bin/shutter line 10732. Variable "$d" will not stay shared at /usr/bin/shutter line 10738. Variable "$sd" will not stay shared at /usr/bin/shutter line 10777. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10801. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10808. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10815. Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10822. Variable "$window" will not stay shared at /usr/bin/shutter line 10830. Variable "$nav_toolbar" will not stay shared at /usr/bin/shutter line 10840. Name "Gtk3::Gdk::SELECTION_CLIPBOARD" used only once: possible typo at /usr/bin/shutter line 298. WARNING: gnome-web-photo is missing --> screenshots of websites will be disabled!   at /usr/bin/shutter line 9096.         Shutter::App::fct_init_depend() called at /usr/bin/shutter line 196 WARNING: Image::ExifTool is missing --> writing Exif information will be disabled!   at /usr/bin/shutter line 9115.         Shutter::App::fct_init_depend() called at /usr/bin/shutter line 196 WARNING: AppIndicator is missing --> there will be no icon showing up in the status bar when running Unity!   at /usr/bin/shutter line 9136.         Shutter::App::fct_init_depend() called at /usr/bin/shutter line 196 Subroutine lookup redefined at /usr/share/shutter/resources/modules/Shutter/Draw/DrawingTool.pm line 28.         require Shutter/Draw/DrawingTool.pm called at /usr/bin/shutter line 229 Variable "$self" will not stay shared at /usr/share/shutter/resources/modules/Shutter/Draw/DrawingTool.pm line 679.         require Shutter/Draw/DrawingTool.pm called at /usr/bin/shutter line 229 Variable "$self" will not stay shared at /usr/share/shutter/resources/modules/Shutter/Screenshot/SelectorAdvanced.pm line 840.         require Shutter/Screenshot/SelectorAdvanced.pm called at /usr/bin/shutter line 234 0.99.3 Rev.1607 [misa@E ~]$  

#### Extra information, such as Shutter version, display server in use (Xorg or Wayland), operating system and ideas for how to solve:
idk not even shutter -v works

**Comment by Photon89** [comment]
That's version 0.99.3, which is unfortunately broken. Possibly an update to 0.99.4 solves the crashing issue.

The output you posted is expected, Shutter is quite verbose, unfortunately. But it correctly gave the version in the last line of the output. For debugging the crash, we'd need the output for the actual crash, not for `shutter -v`.

**Comment by Photon89** [comment]
@lasermtv07 Is the issue this present?

**Comment by lasermtv07** [comment]
hi and thanks of ur response. I switched DE and screenshot tool, but if u want, I can try it out tmr. Thanks for ur response

**Comment by Photon89** [comment]
I assume that this is an issue which has been already fixed in 0.99.4, so if it is solved for you, I'll just close it. Feel free to open a new issue if you happen to catch it again!

**Comment by elisdg** [comment]
seems to work now. interestingly (on manjaro) it shows (even after reboot):
![grafik](https://github.com/shutter-project/shutter/assets/15905294/4db5048c-b818-4cdd-a918-3e9b08983bdc)

**Comment by Photon89** [comment]
@elisdg Yep, I don't know how this happened. I merged a PR which bumps the version and created a release one minute afterwards, looks like the changes didn't get picked up...
